app/models/form_submission_attempt.rb

```diff
@@ -25,15 +25,31 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        user_uuid = form_submission.user_account.id
+        form_type = form_submission&.form_type
+        log_info = { form_submission_id:, benefits_intake_uuid: form_submission&.benefits_intake_uuid, form_type:,
+                     user_uuid: user_uuid }
+        zsf_function = "#{form_id} form submission to Lighthouse"
         if should_send_simple_forms_email
-          Rails.logger.info({
-                              message: 'Preparing to send Form Submission Attempt error email',
-                              form_submission_id:,
-                              benefits_intake_uuid: form_submission.benefits_intake_uuid,
-                              form_type: form_submission.form_type
-                            })
+          Rails.logger.info('Preparing to send Form Submission Attempt error email', log_info)
           simple_forms_enqueue_result_email(:error)
+        elsif should_send_form526_form4142_email
+          zsf_function = 'Form 526 Flow - Form 4142 failure email queuing'
```

Comment:
Would this be better assigned to a constant available to the class?

Reply:
+ 1, should make testing easier as well

Reply:
will do

Reply:
addressed in 0d87bc5c86

Reply:
Ignore as the variable is reassigned.